src/docs/commands_keybindings.md

```diff
@@ -42,6 +42,8 @@ export class HelloworldCommandContribution implements CommandContribution {
 }
 
 ```
+To make the command execution context sensitive, the `CommandHandler` can optionally implement `isEnabled` and `isVisible`. Theia will execute the active handler that returns true on `isEnabled`, so there should be only one active handler doing so. `isVisible` controls the visibility of menu items that are connected to a command, if the active handler returns true, the menu item will be visible and vice versa.
```

Comment:
> Theia will execute the active handler that returns true on `isEnabled`, so there should be only one active handler doing so.

I'm not sure what we are referring to here, can you elaborate, or share what you are trying to express?

Reply:
I tried to make it more clear